cmd/minishift/cmd/start_preflight.go

```diff
@@ -50,13 +51,20 @@ func preflightChecksBeforeStartingHost() {
 	driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic (https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html) for more information"
 	prerequisiteErrorMessage := "See the 'Installing Prerequisites for Minishift' topic (https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites) for more information"
 
-	preflightCheckSucceedsOrFails(
-		configCmd.SkipCheckOpenShiftRelease.Name,
-		checkOriginRelease,
-		fmt.Sprintf("Checking if requested OpenShift version '%s' is valid", viper.GetString(configCmd.OpenshiftVersion.Name)),
-		configCmd.WarnCheckOpenShiftRelease.Name,
-		fmt.Sprintf("%s is not a valid OpenShift version", viper.GetString(configCmd.OpenshiftVersion.Name)),
-	)
+	if checkInternetConnectivity() {
```

Comment:
maybe use a WARN level, just like the ping test does?

See default value for WarnCheckNetworkPing. this might be a shorter solution to get a FAIL.
and implement a specific test for handling the check for github,com, docker hub, and others... to indicate connectivity more clearly... but not as part of the openshift test directly.

WDYT?

Reply:
This way, when offline is initiated, we can just SKIP this specific test... but otherwise they would just pass with WARN.

Reply:
I also do not think `Dial` handles proxy setups?

Reply:
@gbraad Will check that.